Resources/Prototypes/_Sunrise/tts_voices_sunrise.yml

```
@@ -893,15 +893,21 @@
 - type: ttsVoice
   id: HeartstoneBob
   name: tts-voice-name-heartstone-bob1
-  sex: Male
+  #Lust-Start
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)